packages/driver/cypress/integration/e2e/multi-domain/multi_domain_cypress_api.spec.ts

```diff
@@ -0,0 +1,290 @@
+// @ts-ignore / session support is needed for visiting about:blank between tests+
+describe('multi-domain Cypress API', { experimentalSessionSupport: true, experimentalMultiDomain: true }, () => {
+  beforeEach(() => {
+    cy.visit('/fixtures/multi-domain.html')
+    cy.get('a[data-cy="multi-domain-secondary-link"]').click()
+  })
+
+  context('Commands', () => {
+    context('add', () => {
+      it('adds a custom command', () => {
+        cy.switchToDomain('foobar.com', () => {
+          // @ts-ignore
+          Cypress.Commands.add('foo', () => {
+            return cy.wrap('bar')
+          })
+
+          // @ts-ignore
+          cy.foo().then((shouldBeBar) => {
+            expect(shouldBeBar).to.equal('bar')
+          })
+        })
+      })
+
+      it('persists defined commands through spec bridge', () => {
+        cy.switchToDomain('foobar.com', () => {
+          // @ts-ignore
+          cy.foo().then((shouldBeBar) => {
+            expect(shouldBeBar).to.equal('bar')
+          })
+        })
+      })
+    })
+
+    context('overwrite', () => {
+      it('overwrites an existing command in the spec bridge', () => {
+        cy.switchToDomain('foobar.com', () => {
+          // @ts-ignore
+          Cypress.Commands.overwrite('foo', () => {
+            return cy.wrap('baz')
+          })
+
+          // @ts-ignore
+          cy.foo().then((shouldBeBaz) => {
+            expect(shouldBeBaz).to.equal('baz')
+          })
+        })
+      })
+
+      it('persists overwritten command through spec bridge', () => {
+        cy.switchToDomain('foobar.com', () => {
+          // @ts-ignore
+          cy.foo().then((shouldBeBaz) => {
+            expect(shouldBeBaz).to.equal('baz')
+          })
+        })
+      })
+    })
+  })
+
+  context('Keyboard', () => {
```

Comment:
I believe we decided we don't want defaults inherited in with any of the `defaults` methods, because some of them take functions and it would be inconsistent to support some but not other or some properties but not others. 

Only `Cypress.config()` is inherited in, because it should only have serializable values.

Reply:
That's right 👍